src/components/MatchedUsers.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+
+  <el-card>
+    <template #header>
+    <h1>People</h1>
+    </template>
+    <li
+        v-for="(user, index) in users"
+        :key="`message:${index}-${user}`"
+    >
+      <el-divider v-if="index !== 0"/>
+      <el-row align="middle" type="flex" v-on:click="OpenUrlInNewTab(user.html_url)" class="clickable">
+        <el-avatar class="avatar" :src="user.avatar_url" />
+        {{ user.login }}
+      </el-row>
+    </li>
+  </el-card>
+</template>
+
+<script>
+
+import {mapGetters} from "vuex";
+
+export default {
+  name: "MatchedUsers",
+  props: [
+      "id"
+  ],
+  computed: {
+    ...mapGetters("adoptableChat", ["users"]),
+  },
+  methods: {
+    OpenUrlInNewTab(url) {
+      window.open(url)
+    }
+  }
+}
+</script>
+
+<style scoped>
+.avatar {
+  margin-right: 10px;
+}
+.clickable {
```

Comment:
if you want to you can make a clickable class in the default.scss file, its available everywhere